webpack.config.js

```diff
@@ -16,6 +17,11 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( {
 			injectPolyfill: true,
 		} ),
+		new webpack.DefinePlugin( {
+			__PAYMENT_METHOD_FEES_ENABLED: JSON.stringify(
```

Comment:
This will be injected in the app at build time, depending on how the application was started. In production build, since `__PAYMENT_METHOD_FEES_ENABLED` will not be `true` by default, all code checking for it to be `true` will be stripped out thanks to tree shaking.